README.md

```diff
@@ -64,6 +64,7 @@ AvatarGlow(
   glowShape: BoxShape.circle,
   animate: _animate,
   curve: Curves.fastOutSlowIn,
+  glowRepeatCount: 5   // Animation will be repeated 5 times
```

Comment:
Here I would keep it the same as the other parameters. Either comment on all of them or leave out the comment